internal/ansible/events/log_events.go

```diff
@@ -62,22 +64,31 @@ func (l loggingEventHandler) Handle(ident string, u *unstructured.Unstructured,
 		"job", ident,
 	)
 
+	verbosityAnnotation, verbosityEnvVar := GetVerbosity(u, e, ident)
+
 	// logger only the following for the 'Tasks' LogLevel
 	t, ok := e.EventData["task"]
 	if ok {
 		setFactAction := e.EventData["task_action"] == eventapi.TaskActionSetFact
 		debugAction := e.EventData["task_action"] == eventapi.TaskActionDebug
 
+		if verbosityAnnotation > 0 || verbosityEnvVar > 0 {
+			l.mux.Lock()
+			logger.Info("[Ansible StdOut Logs]", "EventData.TaskArgs", e.EventData["task_args"])
+			fmt.Println(e.StdOut)
+			l.mux.Unlock()
+			return
+		}
 		if e.Event == eventapi.EventPlaybookOnTaskStart && !setFactAction && !debugAction {
 			l.mux.Lock()
-			logger.Info("[playbook task]", "EventData.Name", e.EventData["name"])
+			logger.Info("[playbook task start]", "EventData.Name", e.EventData["name"])
 			l.logAnsibleStdOut(e)
 			l.mux.Unlock()
 			return
 		}
-		if e.Event == eventapi.EventRunnerOnOk && debugAction {
+		if e.Event == eventapi.EventRunnerOnOk {
```

Comment:
let's actually leave `debugAction` in the conditional, that way if they don't set the annotation or envvar absolutely nothing changes.